<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConvOptionsMono</class>
 <widget class="QWidget" name="WidgetConvOptionsMono">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Byte order:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonBigEndian">
       <property name="text">
        <string>Big Endian</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonLittleEndian">
       <property name="text">
        <string>Little Endian</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData8">
       <property name="text">
        <string>8 bit</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData16">
       <property name="text">
        <string>16 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonData32">
       <property name="text">
        <string>32 bit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMirrorBits">
     <property name="text">
      <string>Mirror bytes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>75</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Convert to monochrome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxConvType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollBar" name="horizontalScrollBar">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonBigEndian</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData8</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData16</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonData32</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMirrorBits</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonLittleEndian</sender>
   <signal>toggled(bool)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxConvType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>WidgetConvOptionsMono</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePreview()</slot>
 </slots>
</ui>
